lib/manager/util/version.js

```diff
@@ -35,3 +40,21 @@ export function versionHasExpired (version: FeedVersion): boolean {
 export function versionHasNotBegun (version: FeedVersion): boolean {
   return moment(version.validationResult.startDate).isAfter(moment())
 }
+
+// Error types that should prevent publishing a feed version to MTC. These types
+// must correspond to those found at:
+// https://github.com/conveyal/gtfs-lib/blob/dev/src/main/java/com/conveyal/gtfs/error/NewGTFSErrorType.java
+export const BLOCKING_ERROR_TYPES = [
+  'ILLEGAL_FIELD_VALUE',
+  'TABLE_IN_SUBDIRECTORY',
+  'WRONG_NUMBER_OF_FIELDS',
+  'MISSING_COLUMN',
+  'TABLE_MISSING_COLUMN_HEADERS',
+  'REFERENTIAL_INTEGRITY'
+]
+
+export const versionsComparator = (a: FeedVersion, b: FeedVersion) => {
```

Comment:
Could have a more specific name indicating it's comparing the updated attribute.